app/src/main/java/com/github/geohunt/app/ui/components/navigation/NavigationController.kt

```diff
@@ -217,7 +217,7 @@ fun NavigationController(
             ClaimChallenge(
                 cid = cid,
                 onFailure = onFailure,
-                onClaimSubmitted = {
+                onSuccess = {
                     navController.popBackStack()
                     navController.navigate("challenge-view/$cid")
```

Comment:
Maybe reference to HiddenRoute.ChallengeView if it exists? or create it! Maybe making the $cid as an argument to the enum could be possible? Idk. Just make it `${HiddenRoute.ChallengeView.route}/$cid` ?

Reply:
Sure !!